challenge_scoring/metrics/scoring.py

```diff
@@ -43,13 +46,17 @@ def _prepare_gt_bundles_info(bundles_dir, bundles_masks_dir,
             raise ValueError(
                 "Missing basic bundle attribs for {0}".format(bundle_f))
 
+        orig_sft = load_tractogram(
+            os.path.join(bundles_dir, bundle_f), ref_anat_fname,
+            bbox_valid_check=False, trk_header_check=False)
+        orig_sft.to_vox()
+        orig_sft.to_center()
+
```

Comment:
Just to inform you...

The ground truth's bundle masks are not the same as if we use scil_compute_streamline_density_maps. They were computed in 2015, before JC changed the code in this repo in 2017. Just so you know, because now I can't compare properly with the upcoming new version of the tractometer. I will re-run the results with updated bundle mask. 

To prove this, if here you add a line bundle_mask = _create_binary_map(orig_sft, ref), you can see that the voxel count is NOT the same as in the ground truth bundle masks.

Reply:
I think it's fine, especially since you cleaned up the bundles which I'm sure impacts their volumes anyways.